build.gradle

```diff
@@ -88,6 +102,36 @@ configurations
     integrationTestRuntime.extendsFrom testRuntime
 }
 
+protobuf {
+
+    generatedFilesBaseDir = 'gen/' 
+
+    protoc {
+        artifact = 'com.google.protobuf:protoc:2.6.1'
```

Comment:
I think it would make sense to use the version from `dependencies.gradle` here for consistency.

Reply:
Currently investigating the proper syntax for this.